perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -36,7 +36,7 @@
   <group>
     <push-ros-namespace namespace="euclidean"/>
     <include file="$(find-pkg-share euclidean_cluster)/launch/voxel_grid_based_euclidean_cluster.launch.xml">
-      <arg name="output/clusters" value="lidar/clusters"/>
+      <arg name="output_clusters" value="lidar/clusters"/>
```

Comment:
launch argument name is defined as `output_clusters` in voxel_grid_based_euclidean_cluster.launch.xml

https://github.com/tier4/AutowareArchitectureProposal.iv/blob/6726f76518dc7c0c19ec4481ec256f6c773b9ad8/perception/object_recognition/detection/euclidean_cluster/launch/voxel_grid_based_euclidean_cluster.launch.xml#L6